services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,

```

Comment:
Use Optional[list[int]] because None as default provided
Fix in all places with all types